test/index.js

```diff
@@ -20,4 +20,30 @@ describe('analytics', function () {
       assert(a.name == b.name);
     });
   });
+
+  describe('noConflict', function () {
+    beforeEach(function () {
+      analytics = window.analytics;
+    });
+
+    afterEach(function () {
+      window.analytics = analytics;
+    });
+
+    it('should leave window.analytics unchanged', function () {
+      // Overwrite window.analytics with random object
+      var dummy = {'dummy': true};
```

Comment:
can you remove the quotes here? we usually go with `{ dummy: true }` for single line objects.